src/app/app-routing.module.ts

```diff
@@ -10,11 +10,16 @@ import { ActivityGraphComponent } from './pages/activity-graph/activity-graph.co
 import { DelegateMonitorComponent } from './pages/delegate-monitor/delegate-monitor.component';
 import { TopAccountsComponent } from './pages/top-accounts/top-accounts.component';
 import { VotersComponent } from './pages/address/voters/voters.component';
+import { TransactionListComponent } from './pages/transaction-list/transaction-list.component';
 
 const appRoutes: Routes = [
   { path: '', component: ExplorerComponent, pathMatch: 'full' },
   { path: 'blocks/:page', component: BlockListComponent },
+  { path: 'transactions/:page', component: TransactionListComponent },
   { path: 'address/:id', component: AddressComponent },
+  { path: 'address/:id/transactions/:type/:page', component: AddressComponent },
```

Comment:
Is there a reason this line exists 3 times?

Reply:
Yes. Code-blindness ;)